tests/posix/xsi_realtime/src/mqueue.c

```diff
@@ -271,3 +271,30 @@ ZTEST(xsi_realtime, test_mqueue_notify_errors)
 	zassert_ok(mq_close(mqd), "Unable to close message queue descriptor.");
 	zassert_ok(mq_unlink(queue), "Unable to unlink queue");
 }
+
+ZTEST(xsi_realtime, test_mqueue_open_and_unlink_multiple)
+{
+	const char *q1 = "q1";
+	const char *q2 = "q2";
+
+	mqd_t mqd1, mqd2;
+        struct mq_attr attrs = {
+                .mq_msgsize = MESSAGE_SIZE,
+                .mq_maxmsg = MESG_COUNT_PERMQ,
+        };
+
+	int32_t mode = 0777;
```

Comment:
Not a huge deal, but if you're updating the PR, might as well use a more conservative mode.

```suggestion
	int32_t mode = 0600;
```